godtools/App/Features/Onboarding/Views/OnboardingQuickStartCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  OnboardingQuickStartCell.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/8/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingQuickStartCell: UITableViewCell {
+    
+    static let nibName: String = "OnboardingQuickStartCell"
+    static let reuseIdentifier: String = "OnboardingQuickStartCellReuseIdentifier"
+        
+    private var item: OnboardingQuickStartItem?
+    
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var subtitleLabel: UILabel!
+    
+    override func awakeFromNib() {
+        
+        super.awakeFromNib()
+    }
+    
+    override func prepareForReuse() {
+        
+        super.prepareForReuse()
+        
+        titleLabel.text = nil
+        subtitleLabel.text = nil
+    }
+    
+    func configure(item: OnboardingQuickStartItem) {
+        
+        self.item = item
+        
```

Comment:
Can you set the ```selectionStyle``` on this cell to ```none```.  By default a color is used which usually doesn't work well with our custom cells.
https://developer.apple.com/documentation/uikit/uitableviewcell/1623221-selectionstyle